tools/pkglint/lib/rules.ts

```diff
@@ -231,6 +232,84 @@ export class ReadmeFile extends ValidationRule {
   }
 }
 
+/**
+ * Headings in the README.md file must be consistent. Concretely, this means the very first heading found in the file
+ * must be at H1, and subsequent headings are required to be H2 or deeper. The document is not allowed to skip a heading
+ * level (i.e: an H4 nested under and H2 - that should have been an H3), nor to have another H1 heading.
+ *
+ * Those validations help ensure a smooth and consistent rendering in the documentation site, especially in the Python
+ * reference documentation.
+ */
+export class ReadmeFileHeadings extends ValidationRule {
+  public readonly name = 'package-info/README.md/Headings';
```

Comment:
How about adding a markdown linter that checks and fixes this?

If you choose, say markdownlint, a combination of [MD001](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md001) and [MD041](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041) is what you're looking for.

I can think of more rules and consistency we need to bring across our construct library READMEs that a linter like this can help on.

Reply:
Well now THAT is an interesting idea... I'm a bit bummed it never occurred to me... 🤦🏻‍♂️